model/src/main/kotlin/config/LicenseFindingCuration.kt

```diff
@@ -27,7 +27,8 @@ import com.fasterxml.jackson.databind.util.StdConverter
 import org.ossreviewtoolkit.spdx.SpdxExpression
 
 /**
- * A curation for license findings.
+ * A curation for a license finding. Use it to correct a license finding or to add a license that was not previously
```

Comment:
Adding a finding is not yet supported. But I believe that should be implemented.

Reply:
Oh. I thought it's already supported as we already have:

https://github.com/oss-review-toolkit/ort/blob/4a262680bdf8db006ef897ee9cf2d50aaa9935b4/model/src/main/kotlin/config/LicenseFindingCurationReason.kt#L47-L50

As it's already documented here, how about also keeping it documented here, and just eventually implement it?

Reply:
Agree, makes sense!